src/pages/test-report/components/legal-requirements/__tests__/legal-requirements.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { ComponentFixture, async, TestBed } from '@angular/core/testing';
+// import { By } from '@angular/platform-browser';
+import { LegalRequirementsComponent } from '../legal-requirements';
+import { MockComponent } from 'ng-mocks';
+import { TickIndicatorComponent } from '../../tick-indicator/tick-indicator';
+import { IonicModule } from 'ionic-angular';
+import { testsReducer } from '../../../../../modules/tests/tests.reducer';
+import { StoreModule, Store } from '@ngrx/store';
+import { StoreModel } from '../../../../../shared/models/store.model';
+import {
+  ToggleNormalStart1,
+  ToggleNormalStart2,
+  ToggleAngledStart,
+  ToggleHillStart,
+} from '../../../../../modules/tests/test_data/test-data.actions';
+
+fdescribe('LegalRequirementsComponent', () => {
+  let fixture: ComponentFixture<LegalRequirementsComponent>;
+  let component: LegalRequirementsComponent;
+  let store$: Store<StoreModel>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        LegalRequirementsComponent,
+        MockComponent(TickIndicatorComponent),
+      ],
+      imports: [
+        IonicModule,
+        StoreModule.forRoot({ tests: testsReducer }),
+      ],
+    })
+      .compileComponents()
+      .then(() => {
+        fixture = TestBed.createComponent(LegalRequirementsComponent);
+        component = fixture.componentInstance;
+        store$ = TestBed.get(Store);
+      });
+  }));
+
+  describe('Class', () => {
+    it('should create', () => {
+      expect(component).toBeDefined();
+    });
+
+    it('should dispatch a TOGGLE_NORMAL_START_1 action', () => {
+      const storeDispatchSpy = spyOn(store$, 'dispatch');
```

Comment:
You could initialise this spy once in the `beforeEach` instead of setting up in every test. Also, no need to assign the return value of `spyOn`, you can do `expect(store$.dispatch).toHaveBeenCalled()`.

Reply:
Moved out to `beforeEach`